src/components/Header/_examples.ts

```diff
@@ -2,19 +2,14 @@ export const basic = {
   lineage: `package ship
 
 import "github.com/grafana/thema"
+
 thema.#Lineage
 name: "ship"
-
-seqs: [
-    {
-        schemas: [
-        // v0.0
-            {
-                firstfield: string
-            },
-        ]
-    },
-]`,
+schemas: [{
+	version: [0, 0]
+	schema: firstfield: string
```

Comment:
@joanlopez I've just noticed that this fails on translation, shouldn't the schema be an object - `schema: {firstfield: string}`?

Reply:
I think it's not needed because it's a single field object - they're equivalent.

I tried to run `Validate` and `Translate` with this basic example (translation is only possible from 0.0 to 0.0) as well as with the following one that has two schema versions, where the first is also "inlined" (doesn't look like an object `{...}`) and both worked for me.

Could you share and example and/or steps to reproduce it, please? Thanks!

Reply:
You're right, it works, the error was on my side :)